library/jesd204/axi_jesd204_rx/axi_jesd204_rx_ip.tcl

```diff
@@ -68,8 +68,8 @@ adi_ip_add_core_dependencies { \
   analog.com:user:axi_jesd204_common:1.0 \
 }
 
-set_property display_name "ADI JESD204B Receive AXI Interface" [ipx::current_core]
-set_property description "ADI JESD204B Receive AXI Interface" [ipx::current_core]
+set_property display_name "ADI JESD204C Receive AXI Interface" [ipx::current_core]
```

Comment:
rev C of the standard defines both 8b10b and 64b66b/64b80b encoders, so 204C includes 204B since it is backwards compatible. 
I would rename JESD204 everywhere to JESD204C even the "ADI JESD204 Receive" and "ADI JESD204 Transmit" IPs for consistency to emphasize the core complies with the rev C of the standard.

Reply:
Ok, we can go that route